listenbrainz_spark/hdfs/upload.py

```diff
@@ -46,43 +47,48 @@ def process_json_listens(self, filename, data_dir, tmp_hdfs_path, schema):
             month = filename.split('/')[-1][0:-5]
             dest_path = os.path.join(data_dir, year, '{}.parquet'.format(str(month)))
 
+        if utils.path_exists(dest_path):
```

Comment:
This doesn't make sense in the first read, I figure we should have a parameter passed to the function which tells it to append instead.

Reply:
Added